### DISCUSSION TITLE
Про LanguageWorker_Russian и падежи

### DISCUSSION THREAD

**Discussion by MarisKori** [Ludeon/RimWorld-ru#1381] [Ideas]
Я так понимаю, что этот класс не работает, и не скоро ещё заработает.
А если и заработает, то снова может "отвалиться" с очередным обновлением. Верно?
В связи с этим вопрос:
Почему так происходит и как поспособствовать исправлению?

В особенности интересует, как ввести в игру русские падежи и прочие склонения? Ведь по сути-то нам не так много и надо. Всего лишь определить для каждой сущности в игре все склонения согласно словарю. Для имён существительных - это всего лишь 12 форм (на самом деле далеко не все из них нужны обычно, но лучше вбить с запасом). И далее эти формы можно было бы использовать через тот же {PAWN_что-то} или {0_что-то}.

К слову, если бы я оформлял предложение разрабам, то предложил бы принимать любые формы `<labelXYZ>`, чтобы далее в контексте можно было использовать {PAWN_XYZ} и {0_XYZ}. Это работало бы для любого языка, очевидно. Реализовать такое просто и не займёт много времени. Но я не уверен, где лучше размещать такое предложение, чтобы его рассмотрели наверняка. Лично я с разрабами не знаком, и прихожусь, в целом, никем, увы.

**Comment by Elevator89** [comment]
Нет, не так вы понимаете ).
Класс работает. Падежи для большинства игровых сущностей в словарях определены. Падежные формы доступны с функцией lookup:
```
  <!-- EN: Assign this bed to be owned by a specific {0}. -->
  <CommandBedSetOwnerDesc>Назначить кровать конкретному {lookup: {0}; Case; 2}.</CommandBedSetOwnerDesc>
```
И все эти чудеса могут работать везде, кроме генеримых текстов (надо пнуть разрабов на этот счёт).
Осталось только добавить эти lookup везде, где можно.
